packages/sui-mono/bin/sui-mono-phoenix.js

```diff
@@ -32,76 +38,49 @@ program
   })
   .parse(process.argv)
 
+const {audit, chunk = DEFAULT_CHUNK, progress = true, root} = program
+
+const NPM_CMD = ['npm', ['install', audit ? '--no-audit' : '']]
 const RIMRAF_CMD = [
   require.resolve('rimraf/bin'),
   ['package-lock.json', 'node_modules']
 ]
-const NPM_CMD = ['npm', ['install']]
-let {chunk = DEFAULT_CHUNK, progress} = program
-chunk = Number(chunk)
+const rootExecution = root ? [RIMRAF_CMD, NPM_CMD] : []
+
+const queue = new Queue({concurrency: +chunk})
 
 const executePhoenixOnPackages = () => {
   if (config.isMonoPackage()) {
     return
   }
-  let taskList = config
-    .getScopesPaths()
-    .map(cwd => [
-      [...RIMRAF_CMD, {cwd, stdio: 'ignore'}],
-      [...NPM_CMD, {cwd, stdio: 'ignore'}]
-    ])
-    .map(commands => ({
-      title:
-        'rimraf node_modules && npm i ' +
-        ('@' + basename(commands[0][2].cwd)).grey,
-      task: () => serialSpawn(commands)
-    }))
 
-  const withChunks = !!chunk && taskList.length > chunk
-  if (withChunks) {
-    taskList = splitArray(taskList, chunk).map((group, i) => {
-      const subTitle = progress
-        ? ''
-        : '\n' + group.map(task => task.title).join('\n')
+  const scopes = config.getScopesPaths()
 
-      return {
-        title: `#${i + 1} group of ${group.length} packages...` + subTitle,
-        task: () => new Listr(group, {concurrent: true})
-      }
-    })
-  }
-
-  const tasks = new Listr(taskList, {
-    concurrent: !withChunks,
-    renderer: !progress ? PlainTextRenderer : null
-  })
-  return tasks.run()
-}
-
-class PlainTextRenderer {
-  constructor(tasks, options) {
-    this._tasks = tasks
-  }
+  scopes.map(cwd => {
+    const commands = [
+      [...RIMRAF_CMD, {cwd, stdio: 'ignore'}],
+      [...NPM_CMD, {cwd, stdio: 'ignore'}]
+    ]
 
-  render() {
-    this._tasks.forEach(task =>
-      task.subscribe(event => {
-        if (event.type === 'STATE') {
-          console.log(
-            task.isPending()
-              ? task.title + '\n' + '... work in progress ...'.yellow
-              : '... completed successfully!'.green + '\n'
+    queue
+      .add(() => serialSpawn(commands))
+      .then(() => {
+        if (progress) {
+          const {size, pending} = queue
+          const packageName = basename(cwd).grey
+          logUpdate(
+            `Installed ${packageName}. ${size + pending}/${scopes.length}`
```

Comment:
IMHO, should be "All packages installed"

Reply:
I agree the message could be improved.

With your message would be:
```
button. 23/45 installed
```

Now is:
```Installed button. 23/45```

What  do you think to something like this to make it more clear?
```button installed. 23 of 45 packages installations```

Reply:
`Button installed. Progress: 23 of 45 packages
button: 23 of 45 packages installed`

Reply:
Changed! Thanks @adriadoz